test/issue103-single-file-package.d

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env dub
+/+ SDL
+name "hello_world"
++/
```

Comment:
Good point. While normal dependencies work fine, while extending the test case, I noticed that path based dependencies are broken. Fixing this cleanly requires some more thought.